Backend/BackendAPI/Models/BikeStation.cs

```diff
@@ -8,9 +8,11 @@ namespace BackendAPI.Models
 {
     public class BikeStation
     {
+        //Klucz główny
         public int ID { get; set; }
         public BikeStationState State { get; set; }
         public string LocationName { get; set; }
-        public List<Bike> Bikes { get; set; }
+
+        public ICollection<Bike> Bikes { get; set; }
```

Comment:
jak chcemy mieć interfejsy zamiast klas (propsuje) to dałbym IList zamiast ICollection bo daje nam to dostęp do metod typowo lostowych
zignorować jeżeli robi to jakieś problemy z EF

Reply:
ok zobaczę czy to rodzi jakieś problemy